xc/common/cmake/project_ray.cmake

```diff
@@ -607,7 +607,6 @@ function(PROJECT_RAY_TILE_CAPACITY)
   #   ARCH <arch>
   #   TILE <tile>
   #   SITE_TYPE <site type>
-  #   SITE_COORDS [X|Y|XY]
```

Comment:
Comment needs to be updated to match behavior?

Reply:
Done